migrations/5_amend_agreements.js

```diff
@@ -0,0 +1,42 @@
+require("babel-register");
+const controlAccounts = require("./accounts").default;
+
+const Neumark = artifacts.require("Neumark");
+const Commitment = artifacts.require("Commitment");
+
+// Maps roles to accounts on live network
+let PLATFORM_OPERATOR_REPRESENTATIVE;
+const RESERVATION_AGREEMENT =
+  "ipfs:QmPXME1oRtoT627YKaDPDQ3PwA8tdP9rWuAAweLzqSwAWT";
+const NEUMARK_HOLDER_AGREEMENT =
+  "ipfs:QmPXME1oRtoT627YKaDPDQ3PwA8tdP9rWuAAweLzqSwAWT";
+
+module.exports = function deployContracts(deployer, network, accounts) {
+  // do not deploy testing network
+  if (network === "inprocess_test" || network === "coverage") return;
+  [, , , , PLATFORM_OPERATOR_REPRESENTATIVE] = controlAccounts(
+    network,
+    accounts
+  );
+  if (network.endsWith("_live")) {
+    console.log("---------------------------------------------");
+    console.log(
+      `Must use ${PLATFORM_OPERATOR_REPRESENTATIVE} account to deploy agreements on live newtork`
+    );
+    console.log("---------------------------------------------");
+    return;
+  }
+
+  deployer.then(async () => {
```

Comment:
why not `await deployer`

Reply:
please nooo! if you export async function from test to truffle, it will fail in so unpredictable way. we spent hours to find this pattern, which works deterministic. better not touch it now!

Reply:
sure ;D i am fine with this solution if it works reliably